stdlib/LinearAlgebra/src/schur.jl

```diff
@@ -149,27 +149,6 @@ included or both excluded via `select`.
 ordschur(schur::Schur, select::Union{Vector{Bool},BitVector}) =
     Schur(ordschur(schur.T, schur.Z, select)...)
```

Comment:
It looks like there is still a call to the deprecated signature here.

Reply:
Thanks Jeff. I think the deprecation is more involved than I initially thought. The code itself uses the `ordschur` method with individual components in a number of places to define the method with factorization object. 

Do you think it's worth going ahead and rewriting the parts or should I close this PR? A minimum PR would then only correct the wrong documentation (see #28145)

Reply:
You could change the deprecated methods that are still needed to an internal function, `_ordschur`.